openapi.yaml

```diff
@@ -0,0 +1,668 @@
+
+  /v2/service_instances/{instance_id}:
+    put:
+      summary: provision a service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.provision
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id of instance to provision
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous operations supported
+          schema:
+            type: boolean
+      requestBody:
+        description: parameters for the requested service instance provision
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceInstanceProvisionRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceProvision'
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceProvision'
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '422':
+          description: Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    patch:
+      summary: update a service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.update
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id of instance to update
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous operations supported
+          schema:
+            type: boolean
+      requestBody:
+        description: parameters for the requested service instance update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceInstanceUpdateRequest'
+      responses:
+        '200': 
```

Comment:
This is missing the `operation` field that is valid in the response of PATCH on Service Instances

Reply:
ah yes, true. Ref: https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#body-5

Reply:
done.

Reply:
Looking at the spec, for PATCH it says this:

>  200 --> MUST be returned if the request's changes have been applied. The expected response body is {}.

So the spec does not say the `operation` field is allowed.

Reply:
@n3wscott I agree; `operation` shouldn't be returned here as a 200 implies the update has completed synchronously.